verifier/verifier_test.go

```diff
@@ -605,3 +605,32 @@ func TestVerifyX509TrustedIdentities(t *testing.T) {
 		})
 	}
 }
+
+func TestVersionCompare(t *testing.T) {
+	var testMinVer1 string = "0.0.9"
+	var testMinVer2 string = "1.0.1"
+	var testMinVer3 string = "1.0.0"
+	var testMinVer4 string = "1.0.0-alpha"
+	var testMinVer5 string = "1-pre+meta"
+	var testPlugVer string = "1.0.0"
+	val := versionCompare(testMinVer1, testPlugVer)
+	if val != true {
+		t.Errorf("TestVersionCompare Error: %v WantErr: true", val)
+	}
+	val1 := versionCompare(testMinVer2, testPlugVer)
+	if val1 != false {
+		t.Errorf("TestVersionCompare Error: %v WantErr: false", val1)
+	}
+	val2 := versionCompare(testMinVer3, testPlugVer)
+	if val2 != true {
+		t.Errorf("TestVersionCompare Error: %v WantErr: true", val2)
+	}
+	val3 := versionCompare(testMinVer4, testPlugVer)
+	if val3 != true {
+		t.Errorf("TestVersionCompare Error: %v WantErr: true", val3)
+	}
+	val4 := versionCompare(testMinVer5, testPlugVer)
+	if val4 != true {
+		t.Errorf("TestVersionCompare Error: %v WantErr: true", val4)
+	}
```

Comment:
1. looks like this tests are almost duplicate can we reuse code?
2. In case of failure how will output look like? can we make it more readable?

Applicable to all the tests.

Reply:
Modified the code accordingly